CHANGELOG.md

```diff
@@ -147,6 +147,7 @@ Breaking changes:
     - In your `package.json` you can define:
       - `theiaPluginDir`: to specify the folder in which to download plugins, in respect to your `package.json`
       - `theiaPlugins`: to specify the list of plugins in the form of `"id": "url"`
+- [core] added a new React-based dialog type `ReactDialog` [#6855](https://github.com/eclipse-theia/theia/pull/6855)
```

Comment:
We recently released `0.15.0`, do you mind adding the following entry under a new category `0.16.0`?